kitty/index.html

```diff
@@ -0,0 +1,155 @@
+              </div>
+              
+              <div class="box">
+                <h5>Pending for approvals</h5>  
+           
+                <div class="box pending-items">   
+                  <h2>Order Request</h2>
+                  <p >6 </p>
+                </div>
+
+                <div class="box pending-items">
+                  <h2>Expense Request</h2>
+                  <p>6</p>
+                </div>
+
+                <div class="box pending-items">
+                  <h2>Travel Request</h2>
+                  <p>6</p>
+                </div>
+              </div>
+
+              <div class="box">
+                <h5>Recent Comments</h5>
+                <div class="profile-comment">
+                  <div class="profile-photo">
+                    <img src="img/profilephoto.jpg"  height="50" class="clip-circle">
+                  </div>
+                </div>
+                <p>Order #123 Jul 16, 2018, 3:23pm <br> 
+                Purhased by: Cid Karmer</p>
+              </div>   
+            </div>
+           
+            <div class="col-md-6 col-sm-12 col-xs-12 ">
+                <div class="card">
+                    <div class="card-body">
+                      <h5 class="card-title">What's New?</h5>
+                      <p class="card-text">Vestibulum id ligula porta felis euismod semper. Maecenas sed diam eget risus varius blandit sit amet non magna..</p>
+                      <a href="#" class="card-link">Learn more</a>
+                     
+                    </div>
+                  </div>
+             
+              <div class="box recent-activities">
+                <h5>Recent Activities</h5>
+                    
+                <div class="recentcontainer">
+                  <button type="button" class="btn btn-light">
+                  <span class="">RFO</span>
+                  123
+                  </button>
+                      
+                  <div>
+                    <p>Has been Approved </p>
+                    <p>Jul 16, 2018, 3:23pm Purhased by: Cid Karmer</p></div>
+              </div>
+
+              <div class="recentcontainer">
+                    <button type="button" class="btn btn-light">
+                    <span class="">RFO</span>
+                    123
+                    </button>
+                        
+                    <div>
+                      <p>Has been Approved </p>
+                      <p>Jul 16, 2018, 3:23pm Purhased by: Cid Karmer</p></div>
+              </div>
+
+              <div class="recentcontainer">
+                      <button type="button" class="btn btn-light">
+                      <span class="">RFO</span>
+                      123
+                      </button>
+                          
+                      <div>
+                        <p>Has been Approved </p>
+                        <p>Jul 16, 2018, 3:23pm Purhased by: Cid Karmer</p></div>
```

Comment:
For readability try to bump the closing `</div>` tag down so that its easy to see where it starts and ends!

```
<div>
    ...
</div>
```